src/parser.py

```diff
@@ -468,6 +468,12 @@ def parse(my_parser):
         default=2,
         help="\t\tSpecify the decimal to display. (DEFAULT: 2)",
     )
+    analyze_group.add_argument(
+        "--save-dfs",
+        dest="df_file_dir",
+        metavar="",
+        help="\t\tSpecify the dirctory to save analysis dataframe csv files.",
+    )
```

Comment:
I agree with your logic. In the future saving analysis will be helpful, but only when logic is available to to use this saved analysis as cached data to reduce processing time.

Until then, I find the `--save-dfs` is not needed since we already have the `--output` flag to save output data. In the interest of keeping CLI options simple, I'm commenting out.

```suggestion
    # analyze_group.add_argument(
    #    "--save-dfs",
    #    dest="df_file_dir",
    #    metavar="",
    #    help="\t\tSpecify the dirctory to save analysis dataframe csv files.",
    # )
```

Reply:
The scenario of --output is different.  --output saves only the text report, which can be shared to others as overview.
--save-dfs save all the dfs in csv, which can be used for further analysis or other visualization. @arghdos, does this match your expectation?

Reply:
Yes -- I need the raw dataframes quite often, and dumping the pared output (with all of the crazy unicode chars) is not really a substitute.  IMO, these can co-exist.